chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -373,6 +375,7 @@ impl KeyValueRuntime {
             headers_cache: RwLock::new(HashMap::new()),
             state: RwLock::new(state),
             state_size: RwLock::new(state_size),
+            contract_cache: NoContractRuntimeCache,
```

Comment:
should we use MockContractRuntimeCache here instead of a no-op one?

Reply:
We could, but kv runtime doesn't operate on contracts, so it won't ever update the cache. In that case I think the no-op one fits better.